test/index-spec.js

```diff
@@ -114,13 +114,13 @@ describe('Index', async () => {
   });
 
   it('Correctly accept a file', async () => {
-    await fs.writeFile(`${__dirname}/testClient/test.txt`, 'Howdy');
-    const fileStream = fs.createReadStream(`${__dirname}/testClient/test.txt`);
+    await client.testApi.upload({ theFile: 'words' }, {}).should.be.resolvedWith({ content: 'words' });
 
-    await client.testApi.upload({ theFile: 'howdy' }, {}).should.be.rejectedWith({
```

Comment:
this is no longer a rejection cause of a change in bravado client generator that allows handing in strings to file params from a while back